scenes/gameplay/Death.tscn

```diff
@@ -0,0 +1,6 @@
+[gd_scene load_steps=2 format=3 uid="uid://bbintxmi3bxga"]
```

Comment:
Not necessary to create a scene with just a node with a scripts attached. Script can be attached in the scene where it's needed.